posthog/health.py

```diff
@@ -171,17 +167,17 @@ def is_kafka_connected() -> bool:
     return can_connect_to_kafka()
 
 
-def is_postgres_connected() -> bool:
+def is_postgres_connected(db_alias=DEFAULT_DB_ALIAS) -> bool:
     """
     Check we can reach the main postgres and perform a super simple query
 
     Returns `True` if so, `False` otherwise
     """
     try:
-        with connections[DEFAULT_DB_ALIAS].cursor() as cursor:
+        with connections[db_alias].cursor() as cursor:
             cursor.execute("SELECT 1")
-    except DjangoDatabaseError:
-        logger.debug("postgres_connection_failure", exc_info=True)
+    except Exception:
+        logger.exception("postgres_connection_failure", exc_info=True)
```

Comment:
I only found out the problem because a separate healthcheck for decide actually logs at exception level. At debug level, these aren't captured in loki by default.

How do you feel about making these logs exception level instead of debug level for all checks?

Currently I think these failures are rare, and earlier I think the reasoning was that we don't want to also overwhelm loki with log spew when things are down, causing more issues. Not sure how big a problem this is right now?